src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphDAO.scala

```diff
@@ -92,4 +98,94 @@ trait GraphDAO {
   def getEntityVertex(db: Graph, workspaceNamespace: String, workspaceName: String, entityType: String, entityName: String) = {
     getSinglePipelineResult(entityPipeline(db, workspaceNamespace, workspaceName, entityType, entityName))
   }
+
+  import scala.reflect.runtime.universe._
+  import scala.reflect.runtime.{universe=>ru}
+  import scala.annotation.meta.field
```

Comment:
mostly because when I was trying to get it to work it was easier to have them here, but then it seemed nice to have the imports close to where they are used